src/pages/Home.js

```diff
@@ -1,11 +1,28 @@
 import React, { Component } from 'react';
-import { Container } from 'react-bootstrap';
+import { Container, Row, Col } from 'react-bootstrap';
+import { Link } from '@reach/router';
```

Comment:
Oh, not sure how we didn't catch this - wrong router?

Reply:
Fixed